pgml-dashboard/app/templates/notebooks/cell.html

```diff
@@ -1,4 +1,6 @@
-<turbo-frame id="cell_{{ cell.cell_number }}">
+<turbo-frame id="cell_{{ cell.pk }}">
+  {% if cell.deleted_at %}
```

Comment:
If you're filtering these at the query level why do you need this additional check?

Reply:
Also, is there some sort of extension that adds that filter by default everywhere?

Reply:
Because when we delete a frame, Turbo complains that the frame is gone when we re-render the page. TLDR: I haven't figured out how to delete frames so I'm returning an empty frame when deleting it. It's temporary, the frame is gone on page reload / new page load.